README.md

```diff
@@ -51,4 +51,17 @@ conn.close()
 * bigint
 * double precision
 * character varying
-* date
\ No newline at end of file
+* date
+
+
+### Development:
+```bash
+docker-compose -f dev.docker-compose.yaml run dev bash
+pip install --user -e .[dev,e2e]
+pytest tests/
+```
+If you need to test different PostgreSQL and Python version, you can locally modify dev.docker-compose.yaml or use CI script:
```

Comment:
i don't really like the idea of having 2 right ways of running tests.. in my opinion we should:

1. have full-featured local test runner (e.g. using docker-compose)
2. make CI server use follow out way to run tests (https://docs.travis-ci.com/user/docker/#using-docker-compose)